src/client/dfuse/dfuse_main.c

```diff
@@ -109,16 +124,20 @@ main(int argc, char **argv)
 	int			ret = -DER_SUCCESS;
 	int			rc;
 
+	/* The 'daos' command uses -m as an alias for --scv however
+	 * dfuse uses -m for --mountpoint so this is inconsistent
+	 * but probably better than changing the meaning of the -m
+	 * option here.h
+	 */
 	struct option long_options[] = {
```

Comment:
Sorry, I'm a little confused with the latest changes. are we removing the short options (-p -c -s) ? if so why?
and if they are removed, why are they still listed here?
It looks weird to me to have short version for some options, but not for others..

Reply:
The reason they are listed is so the case statement still can use a single character for the check.   As to why they are removed, I'm not sure.

Reply:
For consistency with with the daos command, see the link that Ken posted.  I'm not sure why the daos command doesn't allow short options but if that's the decision then dfuse should be consistent.